csvcubed/tests/behaviour/steps/infojson2csvw.py

```diff
@@ -0,0 +1,103 @@
+import json
```

Comment:
I'd give these files a different name since `infojson2csvw` doesn't exist in this project. `cli` might be an alternative?